client/containers/DashboardOverview/overviewRows/overviewRowSkeleton.scss

```diff
@@ -1,4 +1,11 @@
 @import '../overviewStyles.scss';
+@import '~@blueprintjs/core/lib/scss/variables';
+
+$status-colors: (
+	'pending': $gold4, //#F0B726
+	'accepted': $green2, //#1C6E42
+	'declined': $red2, //#AC2F33
+);
```

Comment:
@idreyn doing this twice (here and also over in `spubHeaderToolBar.scss`) feels icky -- I'm sure there's composability in scss capabilities, but I haven't seen anything like that in old PubPub.

Reply:
I would `@import` one file from the other and then just reuse the variable!

Reply:
converted it to a mixin, stashed that in `overviewStyles.scss`, imported from there. Buh-duh-bing.